src/modules/dashboard/components/Dialogs/ManageDomainsDialog/ManageDomainsDialog.tsx

```diff
@@ -35,11 +35,32 @@ const MSG = defineMessages({
     id: 'dashboard.ManageDomainsDialog.domainPermissionsList',
     defaultMessage: 'administration',
   },
+
+  manageWhitelistTitle: {
+    id: 'dashboard.ManageDomainsDialog.manageWhitelistTitle',
+    defaultMessage: 'Manage address book',
+  },
+  manageWhitelistDescription: {
+    id: 'dashboard.ManageDomainsDialog.manageWhitelistDescription',
+    defaultMessage: 'Add addresses you trust to your colony’s address book',
+  },
+
+  permissionsText: {
+    id: 'dashboard.ManageDomainsDialog.permissionsText',
+    defaultMessage: `You must have the {permissionsList} permissions in the
+      relevant teams, in order to take this action`,
+  },
+
+  manageWhitelistPermissionsList: {
+    id: 'dashboard.ManageDomainsDialog.manageWhitelistPermissionsList',
+    defaultMessage: 'root',
```

Comment:
This is slightly misleading as its not a 'list' of permissions. Just makes the code hard to read if you are not familiar.

Reply:
What do you suggest instead?

Reply:
root?